src/themes/default/components/core/ProductGalleryZoomCarousel.vue

```diff
@@ -63,12 +72,15 @@ export default {
   },
   data () {
     return {
-      carouselTransitionSpeed: 300
+      carouselTransitionSpeed: 300,
+      currentPage: 0,
+      hideImage: null
```

Comment:
`hideImage` is not very descriptive for what this value actually holds since it implies a boolean. Maybe we can come up with something better and more descriptive / make a computed property for checks like in line 31?

Reply:
Good point @filrak . I believe at the beginning this was a boolean but after some issues with that approach it became an integer and I just never renamed it. I updated now to `hideImageAtIndex`, does that work for you?